Added a description of run_id field

Sassy-Tester opened the pull request:
Added run_id field to both trips.txt and stop_times.txt

Sassy-Tester committed:
Added a description of run_id field

Added run_id field to both trips.txt and stop_times.txt

thzinc commented on gtfs/spec/en/reference.md:
```diff
@@ -171,6 +171,7 @@ File: **Required**
 |  `shape_id` | ID referencing `shapes.shape_id` | Optional | Identifies a geospatial shape describing the vehicle travel path for a trip. |
 |  `wheelchair_accessible` | Enum | Optional | Indicates wheelchair accessibility. Valid options are:<br><br>`0` or empty - No accessibility information for the trip.<br>`1` - Vehicle being used on this particular trip can accommodate at least one rider in a wheelchair.<br>`2` - No riders in wheelchairs can be accommodated on this trip. |
 |  `bikes_allowed` | Enum | Optional | Indicates whether bikes are allowed. Valid options are:<br><br>`0` or empty - No bike information for the trip.<br>`1` - Vehicle being used on this particular trip can accommodate at least one bicycle.<br>`2` - No bicycles are allowed on this trip. |
+|  run_id | ID | Optional | Identifies the run to which the trip belongs. A run consists of a single trip or many sequential trips driven by the same driver, defined by shared service days and run_id. A run_id can have trips with different service days, making distinct runs. |
```
For consistency with the rest of the document, make sure that `run_id` is surrounded by \`s, as in \`run_id\`. This formats the text using a mono-spaced font.

stevenmwhite replied:
Also add:
"Mid-trip driver changes are indicated in the `stop_times.txt` file."

thzinc commented on gtfs/spec/en/reference.md:
```diff
@@ -205,7 +206,7 @@ File: **Required**
 |  `shape_dist_traveled` | Non-negative float | Optional | Actual distance traveled along the associated shape, from the first stop to the stop specified in this record. This field specifies how much of the shape to draw between any two stops during a trip. Must be in the same units used in [shapes.txt](#shapestxt). Values used for `shape_dist_traveled` must increase along with `stop_sequence`; t ...
 |  `timepoint` | Enum | Optional | Indicates if arrival and departure times for a stop are strictly adhered to by the vehicle or if they are instead approximate and/or interpolated times. This field allows a GTFS producer to provide interpolated stop-times, while indicating that the times are approximate. Valid options are:<br><br>`0` - Times are considered approximate.<br>`1` or empty - Times ar ...
-
+|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt  to indicate a change in drivers, such as is the case for a mid-trip relief.
```
This line needs a ` |` at the end to properly end the table row. Also, add \`s around `run_id` and `trips.txt`.

Sassy-Tester committed:
Added changes suggested by Daniel James

Sassy-Tester reviewed (commented):
Added changes requested by Daniel James.

Sassy-Tester committed:
Adding Steve's request

 Add "Mid-trip driver changes are indicated in the stop_times.txt file.".

thzinc commented on gtfs/spec/en/reference.md:
```diff
@@ -205,7 +206,7 @@ File: **Required**
 |  `shape_dist_traveled` | Non-negative float | Optional | Actual distance traveled along the associated shape, from the first stop to the stop specified in this record. This field specifies how much of the shape to draw between any two stops during a trip. Must be in the same units used in [shapes.txt](#shapestxt). Values used for `shape_dist_traveled` must increase along with `stop_sequence`; t ...
 |  `timepoint` | Enum | Optional | Indicates if arrival and departure times for a stop are strictly adhered to by the vehicle or if they are instead approximate and/or interpolated times. This field allows a GTFS producer to provide interpolated stop-times, while indicating that the times are approximate. Valid options are:<br><br>`0` - Times are considered approximate.<br>`1` or empty - Times ar ...
-
+|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt  to indicate a change in drivers, such as is the case for a mid-trip relief.  Mid-trip driver changes are indicated in the stop_times.txt file. |
```
"Mid-trip driver changes are indicated in the stop_times.txt file." should be moved to the description in `trips.txt`.

Sassy-Tester committed:
Put Steve's suggestion in the correct place.

I put it in stop_times.txt  instead of trips.txt.

Sassy-Tester committed:
Second round of changes

Add "A run_id value specified for one stop_time does not apply to subsequent stop_times in the same trip. The run_id value must be repeated in each subsequent row for the remainder of the trip following a driver switch.
If the entire trip is performed by a single driver, use the run_id in trips.txt instead."

thzinc commented on gtfs/spec/en/reference.md:
```diff
@@ -206,7 +206,8 @@ File: **Required**
 |  `timepoint` | Enum | Optional | Indicates if arrival and departure times for a stop are strictly adhered to by the vehicle or if they are instead approximate and/or interpolated times. This field allows a GTFS producer to provide interpolated stop-times, while indicating that the times are approximate. Valid options are:<br><br>`0` - Times are considered approximate.<br>`1` or empty - Times ar ...
-|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt  to indicate a change in drivers, such as is the case for a mid-trip relief.  Mid-trip driver changes are indicated in the stop_times.txt file. |
+|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt to indicate a change in drivers, such as is the case for a mid-trip relief.   A run_id value specified for one stop_time does not apply to subsequent stop_times in the same trip. The run_id value must be repeated in each subsequent row for the remainder of the trip following a driver switch.
+If the entire trip is performed by a single driver, use the run_id in trips.txt instead.|
```
Because of the way Markdown works, this needs to be on the line above. Use `<br><br>` to add the appropriate line breaks in the text.

i.e., ` …following a driver switch.<br><br>If the entire trip is…`

Sassy-Tester committed:
Made last round of changes suggested by Steve and Daniel.

Sassy-Tester committed:
Another round of changes.
